--- ../pitch-dark/src/ui.common.a	2018-07-21 10:34:52
+++ src/ui.common.a	2023-05-03 00:39:32
@@ -31,11 +31,11 @@
          !byte 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 
 kStringGlobalTitle
-         !raw  "                                   "
+         !raw  "                                  "
          !byte 16                    ; inverse P
          !raw  "itch "
          !byte 4                     ; inverse D
-         !raw  "ark                                   ",0
+         !raw  "arker                                  ",0
 kStringOK
          !byte $0F                   ; 'O' inverse
          !byte 139,0
